src/domain/authorizations/authorizations.js

```diff
@@ -88,43 +110,72 @@ const forwardAuthorizationMessage = async (headers, transactionRequestId, payloa
       'fspiop-destination': fspiopSource
     }
     fspiopError = ErrorHandler.Factory.reformatFSPIOPError(err)
-    await forwardAuthorizationError(errorHeaders, transactionRequestId, fspiopError.toApiErrorObject(Config.ERROR_HANDLING), childSpan)
+    await forwardAuthorizationError(
+      errorHeaders,
+      transactionRequestId,
+      fspiopError.toApiErrorObject(Config.ERROR_HANDLING),
+      childSpan
+    )
     throw fspiopError
   } finally {
     if (childSpan && !childSpan.isFinished && fspiopError) {
-      const state = new EventSdk.EventStateMetadata(EventSdk.EventStatusType.failed, fspiopError.apiErrorCode.code, fspiopError.apiErrorCode.message)
+      const state = new EventSdk.EventStateMetadata(
+        EventSdk.EventStatusType.failed,
+        fspiopError.apiErrorCode.code,
+        fspiopError.apiErrorCode.message
+      )
       await childSpan.error(fspiopError, state)
       await childSpan.finish(fspiopError.message, state)
     }
   }
 }
 
 /**
- * Forwards PUT authorization errors to destination FSP
+ * Forwards PUT/POST authorization errors to destination FSP
```

Comment:
i think this is only for the PUT

Reply:
forwardAuthorizationMessage is also used to forward POST to PISP party so the auth errors for POST should also be handled